kernel/src/syscall/sysinfo.rs

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use aster_time::read_monotonic_time;
+
+use super::SyscallReturn;
+use crate::prelude::*;
+
+#[derive(Debug, Default, Clone, Copy, Pod)]
+#[repr(C)]
+pub struct sysinfo {
+    uptime: i64,
+    loads: [u64; 3],
+    totalram: u64,
+    freeram: u64,
+    sharedram: u64,
+    bufferram: u64,
+    totalswap: u64,
+    freeswap: u64,
+    procs: u16,
+    totalhigh: u64,
+    freehigh: u64,
+    mem_unit: u32,
+}
+
+pub fn sys_sysinfo(sysinfo_addr: Vaddr, ctx: &Context) -> Result<SyscallReturn> {
+    let info = sysinfo {
+        uptime: read_monotonic_time().as_secs() as i64,
```

Comment:
I think we can add more information here. 
https://github.com/asterinas/asterinas/blob/4316247d82d86e1230f23883c193c26133b12e6a/ostd/src/mm/stat/mod.rs#L12-L21

Reply:
@grief8 Added `totalram` and `freeram`. 

However integration tests `syscall_test_at_exfat_linux` and `smp_syscall_test_mb2` are failed. And this tests failed extremely strange as for me:
- `syscall_test_at_exfat_linux` failed only at test case `MMapTest.NoReserve`, which is not related to this PR (at least directly).
- `smp_syscall_test_mb2` failed with panic while starting `mknod_test` (according to logs) without even starting `SysinfoTest`.

I don't have enough knowledge about Asterinas and its test system to quickly figure out what's gone wrong. And all syscall tests are passed on my machine (as well as in other workflows, e.g. [this](https://github.com/asterinas/asterinas/actions/runs/12305052400/job/34343819474?pr=1698)) so it becomes hard to debug this failures. Could you please explain what is the idea of different syscalls tests?

Reply:
Never mind. This failure has nothing to do with this pull request. It is a random bug that is being fixed. We can rerun the workflow with the help of `boterinas` by commenting `@boterinas rerun` to solve it.

Reply:
@boterinas rerun

Reply:
@grief8 This comment is resolved then?

Reply:
All right.